community-meetings/2021-11-09.md

```diff
@@ -0,0 +1,32 @@
+# Flatcar community call Tuesday, 9th of November, 5:30pm CEST / 9pm IST / 3:30pm GMT / 11:30am EDT / 8:30am PST
+
+- Zoom link: [https://us06web.zoom.us/j/84336116610](https://us06web.zoom.us/j/84336116610)
+  - Meeting ID: 843 3611 6610
+  - Passcode: 444888
+- Youtube live stream / recording: [https://www.youtube.com/watch?v=YP9HnYxepVo](https://www.youtube.com/watch?v=YP9HnYxepVo)
```

Comment:
```suggestion
- Youtube live stream / recording: to be announced 5 minutes before
```

Reply:
OH NO copy+paste :) Thank you, good catch!